### PR TITLE
Sql schema #69

### PR DESCRIPTION
**Pull request by NMcKittie**
Added the sql schema. I manually test this with dummy data before adding. #69

### CODE REVIEW

File: database/schema.sql [hunk 1]
```diff
@@ -0,0 +1,167 @@
+
+
```
Comment (DKWeeto) [inline database/schema.sql]: random empty space at top?

File: database/schema.sql [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    PRIMARY KEY(flood_id),
+    CONSTRAINT fk_loc
+        FOREIGN KEY(loc_id) 
+            REFERENCES location(loc_id),
+    CONSTRAINT fk_severity
+        FOREIGN KEY(severity_level_id) 
+            REFERENCES severity_level(severity_level_id)
+);
+
+
```
Comment (DKWeeto) [inline database/schema.sql]: inconsistent line spacing

File: database/schema.sql [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+DROP TABLE forecast;
+DROP TABLE weather_report;
+DROP TABLE location;
+DROP TABLE user_details;
+DROP TABLE weather_code;
+DROP TABLE alert_type;
+DROP TABLE severity_level;
+DROP TABLE county;
+DROP TABLE country;
+
```
Comment (DKWeeto) [inline database/schema.sql:14]: maybe DROP TABLE ... IF EXISTS suits better?

**Review by DKWeeto — APPROVED** [review]
merge that beast

File: database/schema.sql (hunk 3 above)
Comment (NMcKittie) [reply database/schema.sql:14]: No

**Pull request merged**